Tests/SnapshotTests/Extensions/UIView+Extensions.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Created by Backbase on 8.12.2022.
+//
+
+import UIKit
+import BackbaseDesignSystem
+
+extension UIView {
+
+    func setSizeConstraints(size: CGSize) -> UIView {
+        self.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Omit these selfs from this file, since they are not needed, since the extension is defined on the View, so the self is unambiguous.